Sources/FeaturevisorSDK/Instance.swift

```diff
@@ -115,7 +88,8 @@ public struct InstanceOptions {
         onRefresh: Listener? = nil,
         onUpdate: Listener? = nil,
         refreshInterval: Int? = nil,
-        stickyFeatures: StickyFeatures? = nil
+        stickyFeatures: StickyFeatures? = nil,
+        sessionConfiguration: URLSessionConfiguration = .default
```

Comment:
we are introducing a new option, but I am glad it's optional and already using a default.

Reply:
We should add the ability to customize the `URLSession` which we are using to fetch the datafile which is happening under Featurevisor initialization. I also changed a little bit a way how we init this object in next PR